src/likelihoods/negativebinomial.jl

```diff
@@ -79,12 +63,13 @@ This corresponds to the definition used by [Distributions.jl](https://juliastats
   p(k|successes, f) = \\frac{\\Gamma(k+successes)}{k! \\Gamma(successes)} l(f)^successes (1 - l(f))^k
 ```
 """
-struct NBParamSuccess{T} <: NBParam
+struct NBParamSuccess{T,Tl} <: NegativeBinomialLikelihood
```

Comment:
here we could now (well not NOW now, in the future) specialize on "LinkFor[0,1]"